charts/osm/templates/osm-configmap.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: osm-config
   namespace: {{ .Release.Namespace }}
 data:
-  permissive_traffic_policy_mode: {{ .Values.OpenServiceMesh.enablePermissiveTrafficPolicy | default "false" | quote }}
+  permissive_traffic_policy_mode: {{ .Values.OpenServiceMesh.enablePermissiveTrafficPolicy | default "true" | quote }}
```

Comment:
The mesh will be installed in permissive mode by default, this is the desired behavior for AKS addon hence I made the same change here

Reply:
Can you please describe what the expected behavior is for this flag?

Reply:
This flag basically starts the mesh with the allow all policy i.e. all services in the mesh will be able to talk to one another by default. 

Once a user wants to use the SMI access policy this field needs to be set to false

Reply:
> Once a user wants to use the SMI access policy this field needs to be set to false
Is this field update-able as part of the configmap that does not require a restart of the pod?

Reply:
Yup its updatable and will not require any pod restart

Reply:
I have created this issue https://github.com/Azure/osm-azure/issues/10 to document all the differences.